canonical-websites/www.ubuntu.com
# 2651: Build beta.ubuntu.com secondary nav

## Done

- Built secondary nav according to [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y) and [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2651)
- Minor fixes to 'Download' tab links

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Navigate to a page that has a secondary nav (e.g. /download)
- Check that the secondary nav matches the [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y) and [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2651)
- Make sure you navigate to a page with secondary breadcrumbs (e.g. /download) as well as tertiary breadcrumbs (e.g. /download/server) to check everything looks good and functions properly.
- **Note:** There is a bit of a visual bug with sections with long names, e.g. [here](http://www.ubuntu.com-pr-2771.run.demo.haus/info/release-end-of-life). I feel it's more of an issue with the sitemap/navigation.yaml than the secondary nav build, but feel free to tell me I'm wrong

## Issues

Fixes #2651
